packages/core/src/checkout-buttons/checkout-button-initializer-options.ts

```diff
@@ -1,4 +1,6 @@
 export default interface CheckoutButtonInitializerOptions {
     host?: string;
     locale?: string;
+    storefrontJwtToken?: string;
+    siteLink?: string;
```

Comment:
Why `siteLink` should be created when we can use `host` instead of it? As far as I remember host is equal to site link

Reply:
`host` is BC Control Panel
`siteLink` is the store url that we need to use making the right redirect on confirmation page

Reply:
For production, we may create a new interface for the headless wallet buttons.

Reply:
Do you mean `host` is the canonical URL, i.e.: `https://store-{hash}.mybigcommerce.com`, and `siteUrl` is the  BC Checkout's site URL (not the headless storefront site URL)?

Reply:
@davidchin yes, you are correct